src/pages/Environments.js

```diff
@@ -202,48 +202,43 @@ const Environments = () => {
   const disksByProject = _.groupBy('googleProject', disks)
   const appsByProject = _.groupBy('googleProject', apps)
 
-  const renderBillingProjectApp = app => {
-    const inactive = !_.includes(app.status, ['DELETING', 'ERROR', 'PREDELETING']) &&
-      getCurrentApp(app.appType)(appsByProject[app.googleProject]) !== app
+  const getBillingProjectCell = (billingProject, shouldWarn) => {
     return h(Fragment, [
-      app.googleProject,
-      inactive && h(TooltipTrigger, {
+      billingProject,
+      shouldWarn && h(TooltipTrigger, {
         content: 'This billing project has multiple active cloud environments. Only the most recently created one will be used.'
       }, [icon('warning-standard', { style: { marginLeft: '0.25rem', color: colors.warning() } })])
     ])
   }
-
-  const renderBillingProjectRuntime = runtime => {
-    const inactive = !_.includes(runtime.status, ['Deleting', 'Error']) &&
+  const renderBillingProjectForApps = app => {
+    const shouldWarn = !_.includes(app.status, ['DELETING', 'ERROR', 'PREDELETING']) &&
+      getCurrentApp(app.appType)(appsByProject[app.googleProject]) !== app
+    return getBillingProjectCell(app.labels.saturnWorkspaceNamespace, shouldWarn)
+  }
+  const renderBillingProjectForRuntimes = runtime => {
+    const shouldWarn = !_.includes(runtime.status, ['Deleting', 'Error']) &&
```

Comment:
Is the case really camelcase for runtimes (instead of all caps like it is for apps)?

We really need some constants or case-insensitive helper methods to check statuses. It's really confusing because the disk statuses are camelcase, but the App ones are all uppercase.

Reply:
I double-checked in the debugger, and yep, it's "Deleting…" here. Ugh, the inconsistency is infuriating.

Reply:
💯